src/odfesqlodbc/es_communication.h

```diff
@@ -20,6 +20,7 @@
 // clang-format off
 #include <memory>
 #include <queue>
+#include <future>
```

Comment:
Are you still using this header?

Reply:
yes, that's required for launching thread (std::async).